server/commands/cycle.js

```diff
@@ -0,0 +1,38 @@
+function invokeLaunchCycleAPI(lgJWT) {
+  const baseURL = process.env.NODE_ENV === 'development' ? 'http://game.learnersguild.dev' : 'https://game.learnersguild.org'
+  const mutation = {
+    query: `mutation { launchCycle }`
+  }
+  return graphQLFetcher(lgJWT, baseURL)(mutation)
+    .then(data => data.launchCycle)
+}
+
+function launchCycle(commandInfo) {
+  try {
+    const {lgJWT, lgPlayer} = Meteor.user().services.lgSSO
+    if (!lgJWT || !lgPlayer) {
+      throw new Error('You are not a player in the game.')
+    }
+    notifyUser(commandInfo.rid, '🚀  Initiating Launch... stand by.')
+    invokeLaunchCycleAPI(lgJWT)
+      // unless our API invocation fails, we'll stay quiet because the server will notify
+      // the user via the web socket
+      .then(response => console.log(`[LG SLASH COMMANDS] '/cycle launch' API response: ${response}`))
```

Comment:
Are you intentionally leaving this log statement in place? Seems like this should be commented-out and only for debugging.